user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Visualizations/Available_visualizations/General/Stepper.md

```diff
@@ -21,6 +21,11 @@ To configure the component:
 
    - When a DOM instance is applied, this will be used to display the current state, preceded by the states the instance has traversed until its current state, and followed by the future states it will go through if it follows the normal flow.
 
+     > [!NOTE]
+     >
+     > - Not all DOM instances maintain a history of their traversed states. This behavior is determined by the module. Instances without a history calculate the path from the initial state to the current state as if they followed the standard flow, similar to a happy path.
+     > - The [DOM instance history objects](xref:DOM_DomInstanceHistorySettings) are by default stored asynchronously, which enhances performance but may result in an outdated history when retrieved. In such cases, the path between the incomplete history and the current state is also completed according to the standard flow.
```

Comment:
Maybe we can remove the hyperlink here then as it refers to the same documentation

Reply:
Done!